### PR TITLE
Add syncthing-discosrv 1.0.1

### PR DESCRIPTION
**Pull request by linsui**

### CODE REVIEW

File: bucket/syncthing-discosrv.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        }
+    },
+    "bin": "stdiscosrv.exe",
+    "persist": [
+        "discovery.db",
+        "cert.pem",
+        "key.pem"
+    ],
+    "pre_install": [
+        "foreach ($name in @('cert.pem', 'key.pem')) {",
```
Comment (rasa) [inline bucket/syncthing-discosrv.json]: As `discovery.db` is not in the .zip, it should be created here, otherwise, it won’t be properly persisted.

Comment (linsui) [reply bucket/syncthing-discosrv.json]: It's a directory. It can be created automatically. Should I creat it manully?

**Comment by linsui** [comment]
There is a problem that the stdiscosrv will generate key in the run directory. So if run `stdiscosry` directly it will generate a different key in user directory instead of use the key in scoop directory.

**Comment by linsui** [comment]
Move to main bucket

**Pull request closed without merging**